src/core/lib/security/credentials/external/aws_request_signer.cc

```diff
@@ -17,8 +17,40 @@
 
 #include "src/core/lib/security/credentials/external/aws_request_signer.h"
 
-#include <openssl/hmac.h>
-#include <openssl/sha.h>
+#if COCOAPODS==1
+  #include <openssl_grpc/hmac.h>
+#else
+  #if COCOAPODS==1
+  #include <openssl_grpc/hmac.h>
+#else
+  #if COCOAPODS==1
+  #include <openssl_grpc/hmac.h>
+#else
+  #if COCOAPODS==1
+  #include <openssl_grpc/hmac.h>
```

Comment:
naive question - why are these look identical/repetitive? 😄

Reply:
updated and removed. These are local cocoapod changes that are not part of the PR

Reply:
ahh i see! makes more sense now! 😄